### PR TITLE
Music player robin

### LINKED ISSUE #458: Python Music Player App
Hey I am a GSSoC 21 participant and would like to contribute to this repo

# Aim

- Python GUI based music player using Pygame

# Details

- A Python GUI Based Music Player Which Uses Mutagen, Pygame, and Tkinter Themes where a user can add music to play-list

I would like to work on this issue

### PR DESCRIPTION
**Pull request by robin025**
# Description

I have created a Music App using python's GUI Tkinter and pygame module along with mutagen to support sounds

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [X] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [X] I have performed a self-review of my own code

- [X] I have created a helpful and easy to understand `README.md`
- [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [ ] My changes generate no new warnings
- [X] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- All the images (assets) needs to be in same folder
- Currently, the path of images is not relative in the program and therefore it is not working. Fix this

**Comment by robin025** [comment]
> * All the images (assets) needs to be in same folder
> * Currently, the path of images is not relative in the program and therefore it is not working. Fix this

Okay..

**Comment by robin025** [comment]
@kaustubhgupta fixed the bug and moved all the images to assests

File: Music-Player-App/.gitignore [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
```
Comment (kaustubhgupta) [inline Music-Player-App/.gitignore:1]: This file is not needed, kindly delete it

File: Music-Player-App/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (kaustubhgupta) [inline Music-Player-App/LICENSE:1]: This file is not needed, kindly delete it

File: Music-Player-App/main.py [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+sub_menu = Menu(tool_bar, tearoff=0)  # to remove dashed line from menu
+tool_bar.add_cascade(label='Help', menu=sub_menu)
+sub_menu.add_command(label="About Us ", command=about_us)
+
+
+mixer.init()
+
+# main_window.geometry("600x300")
+main_window.title("Rockerz")
+main_window.iconbitmap(r"assests/rockerz.ico")
```
Comment (kaustubhgupta) [inline Music-Player-App/main.py:196]: There is no file named "rockerz.ico" in the assets folder. Also remove the "r" before the double quotes

File: Music-Player-App/main.py [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+
+current_lenth = ttk.Label(
+    r_top_frame, text="Current Duration : --:--", relief=GROOVE)
+current_lenth.pack()
+
+playlist_box = Listbox(main_window)
+canvas = Frame(right_frame)
+canvas.pack(pady=5)
+
+pic = PhotoImage(file="assests/images/play.png")
```
Comment (kaustubhgupta) [inline Music-Player-App/main.py:231]: The images are still not loading and getting the error. 

You need to give path as "./Music-Player-App/assests..." everywhere you want to load images.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Did you ran this program before pushing changes?

**Comment by robin025** [comment]
> Did you ran this program before pushing changes?

yes @kaustubhgupta , i ran this program and it was working fine on my machine...

**Comment by robin025** [comment]
fixed all the paths to load images

File: Music-Player-App/readme.MD [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Music-Player
+=======
+# Music Player App Using Pygame and Mutagen
```
Comment (kaustubhgupta) [inline Music-Player-App/readme.MD:3]: After this heading, make every subheading in level two (##)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
The code works perfectly now, just a minor readme update:

File: Music-Player-App/readme.MD (hunk 5 above)
Comment (robin025) [reply Music-Player-App/readme.MD:2]: Done...!!!

**Review by kaustubhgupta — APPROVED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**